### DISCUSSION TITLE
GameMode does not exist

### DISCUSSION THREAD

**Discussion by MasterBroNetwork** [Minestom/Minestom#2215] [Help] [answered]
When I try to use GameMode, It shows as red and acts like it does not exist.
The external libraries view in IntelliJ is showing GameMode as a .class file directly rather than how it normally should.

I'm not sure why this is happening, I'm currently using Gradle 8.8 with Java 21.
A few other files are showing the same problem too, I have already tried all of the IntelliJ Repair IDE steps to solve it and nothing has worked.

![image](https://github.com/Minestom/Minestom/assets/60338675/8b5ccb74-9d97-49e9-8558-85d7f9347690)
![image](https://github.com/Minestom/Minestom/assets/60338675/4e5c9617-6090-4767-8a17-5a5ad6508680)

**Comment by MasterBroNetwork** [comment]
So far, The only thing that links these together is that they are `Enum` classes.

**Comment by cosrnic** [comment, marked as answer]
Try updating your ide to the latest version, otherwise show your dependencies

**Reply by MasterBroNetwork** [reply]
![image](https://github.com/Minestom/Minestom/assets/60338675/225daa0b-f5f9-4163-9f2f-a09938ab219a)
I updated my IDE but now Gradle (8.8) is acting like the Minestom dependency does not exist at all, I have tried the usual troubleshooting steps already and nothing has worked.

**Reply by cosrnic** [reply]
try adding 
```
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(21)
    }
}
```
to your build.gradle.kts

**Reply by MasterBroNetwork** [reply]
Thank you so much, The problem has been fixed now.
